src/network/netdev/netdev.c

```diff
@@ -873,8 +873,8 @@ int netdev_load(Manager *manager, bool reload) {
 
         STRV_FOREACH(f, files) {
                 r = netdev_load_one(manager, *f);
-                if (r < 0)
-                        log_error_errno(r, "Failed to load %s, ignoring: %m", *f);
+                if (r < 0 && !ERRNO_IS_PRIVILEGE(r))
+                        return log_error_errno(r, "Failed to load %s: %m", *f);
```

Comment:
maybe keep the log message for the other cases, with 'ignoring', at debug level?

Reply:
I thought so, but `config_parse()` already shows a log message in that case. Hence, it is not necessary, I think.

Reply:
ok, makes sense then, no need to duplicate